src/it/settings.xml

```diff
@@ -22,14 +22,6 @@ under the License.
 <settings xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd"
   xmlns="http://maven.apache.org/SETTINGS/1.1.0">
-  <mirrors>
-    <mirror>
-      <id>mrm-maven-plugin</id>
-      <name>Mock Repository Manager</name>
-      <url>@repository.proxy.url@</url>
-      <mirrorOf>*</mirrorOf>
-    </mirror>
```

Comment:
As mentioned previously, all of my efforts to remove `mrm-maven-plugin` here and in `plugin-pom` have failed, so I think `mrm-maven-plugin` should be retained.

Reply:
I wasn't sure if this block was mandatory: the documentation at https://maven.apache.org/plugins/maven-invoker-plugin/examples/fast-use.html#fast-build-configuration does not mention it.

I've removed this change from the PR, let's see